MIGRATION.MD

```diff
@@ -0,0 +1,267 @@
+5. Kotlin 1.7 is required in order to integrate the SDK. SDK itself is compiled with Kotlin 1.8, so compiler of Kotlin 1.6 and below cannot read SDK classes metadata.
+
+If you have an error like the following:
+
+```
+A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
+Duplicate class kotlin.collections.jdk8.CollectionsJDK8Kt found in modules kotlin-stdlib-1.8.10 (org.jetbrains.kotlin:kotlin-stdlib:1.8.10) and kotlin-stdlib-jdk8-1.7.20 (org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20)
+```
+
+you need to add the following rules to your buildscript (more details [here](https://stackoverflow.com/a/75298544)):
+
+```kotlin
+dependencies {
+    constraints {
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.10") {
+            because("kotlin-stdlib-jdk7 is now a part of kotlin-stdlib")
+        }
+        implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10") {
+            because("kotlin-stdlib-jdk8 is now a part of kotlin-stdlib")
+        }
+    }
+}
+```
+
+You can always refer to our [sample application](https://github.com/DataDog/dd-sdk-android/blob/9c2d460b6b66161efb1252039a82784792958042/sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt) for an example on how to setup the SDK.
+
+## SDK configuration changes
+
+Better SDK granularity is achieved with the extraction of different products into independent modules. This changes the way SDK is configured.
+
+`com.datadog.android.core.configuration.Configuration.Builder` class has the following changes:
+
+* Client token, env name, variant name (default value is empty string), service name (default value is application ID taken from the manifest) should be provided in the constructor.
+* `com.datadog.android.core.configuration.Credentials` class which was containing parameters mentioned above is removed.
+* `logsEnabled`, `tracesEnabled`, `rumEnabled` are removed from the constructor in favour of the individual products configuration (see below).
+* `crashReportsEnabled` constructor argument is removed. You can enable/disable JVM crash reporting by using `Configuration.Builder.setCrashReportsEnabled` method, by default JVM crash reporting is enabled.
+* RUM, Logs, Trace products configuration methods are removed from `Configuration.Builder` in favour of the individual products configuration (see below).
+
+`Datadog.initialize` method has `Credentials` class removed from the list of the arguments.
+
+`com.datadog.android.plugin` package and all related classes/methods are removed.
+
+### Logs product changes
+
+All the classes related to the Logs product are now strictly in the `com.datadog.android.log` package.
+
+In order to be able to use Logs product you need to import the following artifact:
+
+```kotlin
+implementation("com.datadoghq:dd-sdk-android-logs:x.x.x")
+```
+
+Logs product can be enabled in the following way:
```

Comment:
```suggestion
You can enable the Logs product with the following script:
```

Reply:
Not sure if you meant "script" or "command" here

Reply:
maybe `using the following code snippet` works better? It is not quite a script, it is a part of the application code.